verifiers/utils/sigstore_tuf.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"fmt"
+
+	sigstoreTUF "github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+type SigstoreTUFClient interface {
+	GetTarget(target string) ([]byte, error)
+}
+
+// NewSigstoreTUFClient gets a Sigstore TUF client, which itself is a wrapper around the official TUF client.
+func NewSigstoreTUFClient() (*sigstoreTUF.Client, error) {
+	opts := sigstoreTUF.DefaultOptions()
```

Comment:
I'd recommend we add `CacheValidity` to the default so that repeated verifications in a short time window won't result in constant TUF repo refreshes.

Reply:
I want to keep this as the default, because like we've had in other conversations, the TUF spec requires updates every time you fetch. So I'd prefer to be compliant by default. But also we keep an in-memory cache of the target key, so it already does avoid repeated fetches.

Reply:
To confirm, how long is the in-memory cache?